shifthelper/checks.py

```diff
@@ -138,30 +144,36 @@ def all_recent_alerts_acknowledged(checkname):
     user also cannot reach the website, so nothing will be acknowledged.
     So in that case we simply return False as well
     '''
-    try:
-        all_alerts = requests.get(config['webservice']['post-url']).json()
-    except RequestException:
-        log.warning('Could not check acknowledged alerts')
-        return False
+    now = datetime.utcnow()
 
-    if not all_alerts:
-        return False
+    if alerts is None:
+        try:
+            alerts = get_alerts()
+        except RequestException:
+            log.warning('Could not check acknowledged alerts')
```

Comment:
should we log the exception here, in case it ever happens we might be curious why.